src/THREEPreviewNode.js

```diff
@@ -1,6 +1,17 @@
 (function(TextureGen, THREE) {
   'use strict';
 
+var shader = `
```

Comment:
What's this shader used for?

Since THREEPreviewNode extends BaseNode, it doesn't seem like it's used for anything.